include/sdf/Error.hh

```diff
@@ -247,6 +247,10 @@ namespace sdf
     /// \sa Element::SetXmlPath
     public: void SetXmlPath(const std::string &_xmlPath);
 
+    /// \brief It will print the error to sdferr or throw it using
+    /// SDF_ASSERT depending on its ErrorCode.
+    public: void throwOrPrintError();
```

Comment:
If it's a member function, the first letter should be uppercase.

Reply:
Done in https://github.com/gazebosim/sdformat/pull/1220/commits/f6f171594b4e6a7864c696318a961d1cb4bea061.